### ISSUE TITLE
ImageNet-10 dataset RNN-Pooling

### ISSUE THREAD

**Issue by ahdxwg** [Microsoft/EdgeML#281]
Each class is divided into 1300 images for training and 50 images for validation.The data sets I found online only have 1300 training sets per class, how do I get the test sets

**Comment by SachinG007** [comment]
For ImageNet, the validation set is used as the test set. The place where you downloaded ImageNet should have a val_dir folder as well.
